.env.example

```diff
@@ -0,0 +1,17 @@
+DOMAIN="base domain"
+LETS_ENCRYPT_EMAIL="email for lets encrypt"
+
+CLOUDFLARE_EMAIL="for running dns letsencrypt challenge"
+CLOUDFLARE_API_KEY="for running dns letsencrypt challenge, use Global API Key, not Origin CA Key"
+
+DOCKER_USERNAME="user for pulling images from github package registry"
+DOCKER_TOKEN="token for pulling images from github package registry"
+
+# you can use the following command to generate this
+# echo $(htpasswd -nb username password)
+BASIC_AUTH="user:encoded password"
```

Comment:
Basic auth for what? For Traefik? Maybe we should make this one more descriptive.

Reply:
for now it's being used for all auth services (Traefik dashboard, portainer dashboard, mongo ui and Redis ui)

Reply:
yes, for all UIs with basic auth: Traefik, portainer, mongoui, redisui